<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="verticalSpacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="l_count">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Количество</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sb_count">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>366</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="displayIntegerBase">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="l_interval">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Промежуток</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cb_interval">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="b_getStatistics">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Запросить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Статистика</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Документы</string>
              </property>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="l_d_download">
                 <property name="text">
                  <string>Скачано:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="l_d_set_dowload">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="l_d_upload">
                 <property name="text">
                  <string>Загружено:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="l_d_set_upload">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="l_d_addFav">
                 <property name="text">
                  <string>В &quot;избранное&quot;:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="l_d_delFav">
                 <property name="text">
                  <string>Из &quot;избранного&quot;:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="l_d_set_addFav">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="l_d_set_delFav">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="l_d_update">
                 <property name="text">
                  <string>Обновлено:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLabel" name="l_d_set_update">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="l_d_delete">
                 <property name="text">
                  <string>Удалено:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="l_d_set_delete">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Аккаунты</string>
              </property>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="l_a_updateInfo">
                 <property name="text">
                  <string>Обновлений профиля:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="l_a_updateSpecInfo">
                 <property name="text">
                  <string>Изменений админами:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="l_a_create">
                 <property name="text">
                  <string>Создано:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="l_a_activeOff">
                 <property name="text">
                  <string>Отключено:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="l_a_set_updateInfo">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="l_a_set_create">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="l_a_set_activeOff">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="l_a_set_updateSpecInfo">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="l_a_activeOn">
                 <property name="text">
                  <string>Активировано:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="l_a_set_activeOn">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="b_pdfGenerated">
             <property name="text">
              <string>Сгенерировать отчёт в PDF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="l_info">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
